addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureSuspiciousCommentsScanRuleUnitTest.java

```diff
@@ -120,6 +123,26 @@ public void shouldNotAlertOnSuspiciousCommentIsPartOfWordInJavaScriptResponse()
         assertEquals(0, alertsRaised.size());
     }
 
+    @Test
+    public void shouldCreateOneAlertPerSuspiciousComment()
+            throws HttpMalformedHeaderException, URIException {
+
+        // Given
+        String body =
+                "Some text <script>Some Script Element FIXME: DO something \n FIXME: DO something else </script>\nLine 2\n";
+        HttpMessage msg = createHttpMessageWithRespBody(body, "text/javascript;charset=ISO-8859-1");
+
+        assertTrue(msg.getResponseHeader().isText());
+        assertTrue(msg.getResponseHeader().isJavaScript());
+
+        // When
+        scanHttpResponseReceive(msg);
+
+        // Then
+        assertEquals(2, alertsRaised.size());
+        assertEquals(Alert.CONFIDENCE_LOW, alertsRaised.get(0).getConfidence());
```

Comment:
Worth checking both alerts have the expected evidence too.

Reply:
Fixed this and the rest of the comments.